### ISSUE TITLE
[OneNote] Importer not working properly - "File already exists"

### ISSUE THREAD

**Issue by redrabbitrob** [obsidianmd/obsidian-importer#126]
Attachments are not transferred correctly. PDF's and images are not taken over completely or partially. Also, there is a recurring error that the images are already present. The import stops as well. 

![image](https://github.com/obsidianmd/obsidian-importer/assets/147381679/d4cb63ed-9913-41d0-86f3-8df3023c11f4)

The images seem also to be broken after import.

![image](https://github.com/obsidianmd/obsidian-importer/assets/147381679/4c1ace0b-bc8e-4021-a4c2-f00322ae76f8)

**Comment by nocomputeruser** [comment]
I noticed that as well when I was trying to import items from OneNote. Not sure if there is a workaround for modifying all attachment file names in OneNote.

**Comment by vladik-pwnz** [comment]
same problem :(

**Comment by jx3v** [comment]
I have the same issue.

**Comment by 112345brian** [comment]
same

**Comment by lightheaded** [comment]
same. I ended up with 15,000+ 277-byte attachments as a bonus

**Comment by sol3uk** [comment]
same, this is stopping me from moving over

**Comment by pcpp94** [comment]
Same, in my case it stays this way forever.

![Screenshot from 2023-12-14 16-53-14](https://github.com/obsidianmd/obsidian-importer/assets/77921005/02dfe37d-6569-4b8f-a5f4-cfebac886f9f)

**Comment by MD-ThorErik** [comment]
Replicated it right now. It's taking forever to import my OneNote notes that are imports from Evernote clipper. So many messages similar to: 
```
Failed: "Exported image 20231219045006-236.jpeg" because Error: File already exists.
```
And those image files are corrupted. In fact, they are 0-byte files.

**Comment by mrgaalgdornick** [comment]
same!

**Comment by lniass** [comment]
I stopped migrating to Obsidian because I had too many files considered as "existing" that I did not want to lose.

Hoping there is soon a solution.

**Comment by lightheaded** [comment]
Yeah, I've given up and manually synced everything. A huge hurdle in migration, but as I liked other features of Obsidian so much more than OneNote's, then I decided to put in the effort. I suspect this might be a dealbreaker for many others, though.

**Comment by lniass** [comment]
> Yeah, I've given up and manually synced everything. A huge hurdle in migration, but as I liked other features of Obsidian so much more than OneNote's, then I decided to put in the effort. I suspect this might be a dealbreaker for many others, though.

I confirm. I stopped my migration attempt. But waiting to see until fix is deployed.
I just saw a fix is not too far from reaching us!

**Comment by Qvinto1** [comment]
Same issue, OneNote import stalls and does not work. I just paid for a subscription, please fix!

**Comment by tgrosinger** [comment]
This should be resolved in the latest release - [1.6.1](https://github.com/obsidianmd/obsidian-importer/releases/tag/1.6.1)
